pkg/plugin/util/util.go

```diff
@@ -177,6 +177,14 @@ func UpdateSnapshotWithNewStorageClass(config *rest.Config, itemSnapshot *backup
 
 	// update the PVC storage class
 	old := *pvc.Spec.StorageClassName
+	if old == "" {
```

Comment:
Line 179 is not safe. `old := *pvc.Spec.StorageClassName`
Should check if pvc.Spec.StorageClassNam is nil before dereferencing it.
If pvc.Spec.StorageClassNam is nil or it points to an empty string, ...

Reply:
Add check if pvc.Spec.StorageClassName is nil.